### PR TITLE
Refactoring auth examples to use sdk-server/sdk-react

### PR DESCRIPTION
**Pull request by moe-dev**
## Did you add a changeset?

If updating one of our packages, you'll likely need to add a changeset to your PR. To do so, run [`pnpm changeset`](https://pnpm.io/using-changesets#adding-new-changesets). `pnpm changeset` will generate a file where you should write a human friendly message about the changes. Note how this ([example](https://github.com/tkhq/sdk/blob/b409cd06790f011bf939adcf0755499b8e7497ae/.changeset/extra-http-exports.md?plain=1#L1)) includes the package name (should be auto added by the command) along with the type of [semver change (major.minor.patch)](https://semver.org/) (which you should set).

These changes will be used at release time to determine what packages to publish and how to bump their version. For more context see [this comment](https://github.com/tkhq/sdk/pull/67#issuecomment-1568838440).

### CODE REVIEW

**Review by robin-turnkey — APPROVED** [review]
Valor.

File: examples/email-auth/src/pages/api/auth.ts [hunk 1]
```diff
@@ -29,53 +28,34 @@ export default async function auth(
...
-    if (!apiKeyId) {
-      throw new Error("Expected a non-null API key ID!");
+    if (!userId || !apiKeyId) {
+      throw new Error("Expected non-null values for userId and apiKeyId.");
     }
 
     res.status(200).json({
       userId,
       apiKeyId,
-      // This is simple in the case of a single organization
```
Comment (andrewkmin) [inline examples/email-auth/src/pages/api/auth.ts:72]: is this no longer valid?

File: examples/email-auth/src/pages/index.tsx [hunk 2]
```diff
@@ -43,75 +39,57 @@ export default function AuthPage() {
...
-          {
-            curve: "CURVE_SECP256K1",
-            pathFormat: "PATH_FORMAT_BIP32",
-            path: "m/44'/60'/0'/0/0",
-            addressFormat: "ADDRESS_FORMAT_ETHEREUM",
-          },
-        ],
-      },
+      walletName: data.walletName,
+      accounts: [
```
Comment (andrewkmin) [inline examples/email-auth/src/pages/index.tsx:82]: tiny nit: we could pull this accounts config out of `sdk-browser` :)

File: examples/otp-auth/src/pages/index.tsx [hunk 3]
```diff
@@ -95,44 +91,26 @@ export default function AuthPage() {
...
-          {
-            curve: "CURVE_SECP256K1",
-            pathFormat: "PATH_FORMAT_BIP32",
-            path: "m/44'/60'/0'/0/0",
-            addressFormat: "ADDRESS_FORMAT_ETHEREUM",
-          },
-        ],
-      },
+      walletName: data.walletName,
+      accounts: [
```
Comment (andrewkmin) [inline examples/otp-auth/src/pages/index.tsx:104]: same here, but again, a nit

**Review by andrewkmin — APPROVED** [review]
very clean!

File: examples/email-auth/src/pages/index.tsx (hunk 2 above)
Comment (moe-dev) [reply examples/email-auth/src/pages/index.tsx:82]: Would require installing a new package which may get confusing to users as to why we would require sdk-browser for this example? Perhaps it makes sense to export from react as well

Comment (andrewkmin) [reply examples/email-auth/src/pages/index.tsx:82]: sorry, meant `sdk-server`. both server and browser offer the helpers: https://github.com/tkhq/sdk/blob/main/packages/sdk-server/src/turnkey-helpers.ts

**Pull request merged**